projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/switch-toggle-docs/switch-toggle-docs.component.html

```diff
@@ -0,0 +1,227 @@
+      <p class="go-body-copy">
+        Form hints exist to help give more information about the
+        toggles they are attached to. The <code class="code-block--inline">@Input() key: Array&lt;string&gt;</code>
+        can be used to pass in an array of hints to the component.
+      </p>
+      <div class="go-container">
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">View</h2>
+          <go-switch-toggle [control]="control4"
+                            label="Hints"
+                            [hints]="hints">
+          </go-switch-toggle>
+        </div>
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">Code</h2>
+          <code
+            [highlight]="hintsExTs"
+            class="code-block--no-bottom-margin"
+          ></code>
+          <code
+            [highlight]="hintsExHtml"
+            class="code-block--no-bottom-margin"
+          ></code>
+        </div>
+      </div>
+    </ng-container>
+  </go-card>
+  <go-card id="form-switch-errors" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h1 class="go-heading-5">Component Errors</h1>
+    </ng-container>
+    <ng-container go-card-content>
+      <p class="go-body-copy">
+        FormControls all have a <a class="go-link" href="https://angular.io/api/forms/AbstractControl#setErrors">consistent way of setting errors</a>
+        via the <code class="code-block--inline">setErrors()</code> function.
+        All goponents use this API to display errors on our components, but the
+        data needs to be displayed in a specific structure.
+      </p>
+      <p class="go-body-copy">
+        By default the error type is set to "Error", however this can be
+        overridden as shown below.
+      </p>
+      <div class="go-container">
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">View</h2>
+          <go-switch-toggle [control]="control5"
+                            label="Errors Toggle">
+          </go-switch-toggle>
+        </div>
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">Code</h2>
+          <code
+            [highlight]="errorsExTs"
+            class="code-block--no-bottom-margin"
+          ></code>
+          <code
+            [highlight]="errorsExHtml"
+            class="code-block--no-bottom-margin"
+          ></code>
+        </div>
+      </div>
+    </ng-container>
+  </go-card>
+  <go-card id="switch-label-position" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h1 class="go-heading-5">Label Position</h1>
+    </ng-container>
+    <ng-container go-card-content>
+      <p class="go-body-copy">
+        By default the label for the switch toggle appears after the toggle.
+        If there is a case where the label needs to be positioned before the swith,
```

Comment:
Should this be "switch" rather than "swith"?

Reply:
Yes it should be! Good catch.